config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  root "static_pages#home"
+  get  "/home", to: "static_pages#home"
+  get  "/help", to: "static_pages#help"
+  get  "/contact", to: "static_pages#contact"
+  get  "/signup", to: "users#new"
```

Comment:
```suggestion
  root "static_pages#home"
  
  get "/home", to: "static_pages#home"
  get "/help", to: "static_pages#help"
  get "/contact", to: "static_pages#contact"
  get "/signup", to: "users#new"
```
thẳng hàng là đẹp đấy nhưng đúng indent sẽ ok hơn nhé =))